### ISSUE TITLE
Password entry appears to have a character limit

### ISSUE THREAD

**Issue by rosseloh** [unpoller/unpoller#743]
I just installed the docker image and was having a heck of a time getting the app to authenticate with my controller; I was following the instructions in the docs exactly, apart from adding my custom password. However I was getting authentication errors the entire time. I did have the correct user account created on the controller and was using the correct URL as per the docs. (The command in question was `docker run -e "UP_UNIFI_DEFAULT_URL=https://X.X.X.X" -e "UP_UNIFI_DEFAULT_PASS=y7y7Evf6pGGmmsos" ghcr.io/unpoller/unpoller` - I've redacted the IP per company guidelines but left the password unredacted as it is different now - for example's sake)

Turns out, that password `y7y7Evf6pGGmmsos` either mysteriously has a weird special character despite being plain alphanumerics, or is too long for either unpoller or Unifi itself. I'm leaning towards the latter. Whether that's a bug in unpoller or in the Unifi API, I can't say yet, but the Unifi console lets me use that password just fine on the regular web form. This is also not the only password I tried, but I was generating them with a password manager so it was defaulting to 16 characters.

Sadly haven't had time to test exactly how many characters is too many, but I changed the password to a new one of only 12 characters (Unifi's minimum in the console) and it worked after that.

**Comment by platinummonkey** [comment]
as far as I know unifi has a default minimum of 12 characters with 1 lower alpha, 1 upper alpha and 1 number. The max is 60 chars but this varies by device and thread post it appears. These are read purely as a string, perhaps the UI was originally truncating the longer password?

**Comment by rosseloh** [comment]
I couldn't say for sure since the console doesn't have a "show the hidden characters" button like some password forms do. All I do know is that I was able to successfully log in with the longer passwords via the console itself, but not with unpoller until I shortened it.

If it's not on this end, fair enough! Figured it was worth mentioning since I spent a good hour seeing similar threads with no resolutions posted in the last year (mostly on Reddit, not here).
